packages/compass/scripts/patterns/linux_tar-patterns.json

```diff
@@ -0,0 +1,48 @@
+[
+  "APP-SUFFIX",
+  "APP-SUFFIX/LICENSE",
+  "APP-SUFFIX/LICENSES.chromium.html",
+  "APP-SUFFIX/APP",
+  "APP-SUFFIX/THIRD-PARTY-NOTICES.md",
+  "APP-SUFFIX/chrome-sandbox",
+  "APP-SUFFIX/chrome_100_percent.pak",
+  "APP-SUFFIX/chrome_200_percent.pak",
+  "APP-SUFFIX/chrome_crashpad_handler",
+  "APP-SUFFIX/icudtl.dat",
+  "APP-SUFFIX/libEGL.so",
+  "APP-SUFFIX/libGLESv2.so",
+  "APP-SUFFIX/libffmpeg.so",
+  "APP-SUFFIX/libvk_swiftshader.so",
+  "APP-SUFFIX/libvulkan.so.1",
+  "APP-SUFFIX/locales",
+  "APP-SUFFIX/locales/*.pak",
+  "APP-SUFFIX/resources",
+  "APP-SUFFIX/resources.pak",
+  "APP-SUFFIX/resources/app.asar",
+  "APP-SUFFIX/resources/app.asar.fully-unpacked",
+  "APP-SUFFIX/resources/app.asar.fully-unpacked/build",
+  "APP-SUFFIX/resources/app.asar.fully-unpacked/build/**/*",
```

Comment:
Don't have a strong preference, not sure if it's needed, but maybe we want to limit it to some known extensions to make sure nothing that we don't expect to bundle got in the build

Reply:
Makes sense. I tightened up the patterns.